Taskfile.yaml

```diff
@@ -155,27 +170,52 @@ tasks:
       - task: basic
         vars: { ANSIBLE_PLAYBOOK: "provision-tls-cluster.yml" }
 
-  ci-standup:
-    desc: runs all steps for demo cluster standup
-    vars:
-      TF_CLI_ARGS: '{{ default "" .TF_CLI_ARGS}}'
+  create-tiered-storage-cluster:
+    desc: run multiple ansible playbooks to create a cluster with tiered storage and tls
+    summary: |
+      This command assumes you have the infrastructure in place and does not build it for you.
+      Ensure you have a hosts file in the artifacts directory with name hosts_[CLOUD_PROVIDER]_[DEPLOYMENT_ID].ini
+    deps:
+      - install-role-requirements
+    run: always
+    env:
+      DA_AWS_ACCESS_KEY_ID: '{{ .DA_AWS_ACCESS_KEY_ID | default nil }}'
+      DA_AWS_SECRET_ACCESS_KEY: '{{ .DA_AWS_SECRET_ACCESS_KEY | default nil }}'
+    preconditions:
+      - sh: "[ {{.DA_AWS_ACCESS_KEY_ID}} ]"
+        msg: DA_AWS_ACCESS_KEY_ID envvar not set.
+      - sh: "[ {{.DA_AWS_SECRET_ACCESS_KEY}} ]"
+        msg: DA_AWS_SECRET_ACCESS_KEY envvar not set.
     cmds:
-      - task: keygen
-      - task apply -- '{{.TF_CLI_ARGS}}'
-      - task: create-tls-cluster
+      - task: basic
+        vars: {
+          ANSIBLE_PLAYBOOK: "provision-tls-cluster.yml",
+          ANSIBLE_ARGS: '-e cloud_storage_access_key={{.DA_AWS_ACCESS_KEY_ID}} -e cloud_storage_secret_key={{.DA_AWS_SECRET_ACCESS_KEY}}'
+        }
 
   install-rpk:
     desc: installs rpk
     cmds:
       - mkdir -p {{.ARTIFACT_DIR}}/tmp
       - curl -L https://github.com/redpanda-data/redpanda/releases/latest/download/rpk-linux-amd64.zip -o {{.ARTIFACT_DIR}}/tmp/rpk-linux-amd64.zip
       - mkdir -p {{.ARTIFACT_DIR}}/bin
+      - rm -f {{.ARTIFACT_DIR}}/bin/rpk
       - unzip {{.ARTIFACT_DIR}}/tmp/rpk-linux-amd64.zip -d {{.ARTIFACT_DIR}}/bin/
```

Comment:
**[nit, not worth giving it more like 1 min of testing]:**

I believe you can make unzip override the rpk bin by passing the `-o -f` flags:

```
 -o     overwrite existing files without prompting.  This is a dangerous option, so use it with care.  (It is often used with -f, however, and is the only way to overwrite directory EAs under OS/2.)
 -f     freshen  existing  files, i.e., extract only those files that already exist on disk and that are newer than the disk copies.  By default unzip queries before overwriting, but the -o option may be
        used to suppress the queries. <... more>
```